benefits/tech_scheme.md

```diff
@@ -0,0 +1,19 @@
+# Tech-scheme
+
+We are signed up to Techscheme, part of Cyclescheme, which you can upon successful completion of your probation.
+
+Techscheme lets you get the latest tech worth up to £2,000. You then spread the cost with payments from your salary across 12 or 24 months and make a National Insurance saving of up to 12% too. 
+Best of all – you own the products from the beginning and there’s no end of hire fee.
+
+You can calculate your savings by using their [calculator](https://www.techscheme.co.uk/how-it-works)
+
+# Process for buying tech products
```

Comment:
This heading should start with two hashes ('##') because it's a secondary header.

Reply:
done